libs/execution_context/ebpf_program.c

```diff
@@ -53,9 +53,12 @@ typedef struct _ebpf_program
     const ebpf_extension_data_t* info_extension_provider_data;
     // Program type specific helper function count.
     uint32_t provider_helper_function_count;
+    // Global helper function count implemented by the extension.
+    uint32_t global_helper_function_count;
     bool invalidated;
 
     ebpf_trampoline_table_t* trampoline_table;
```

Comment:
```suggestion
    ebpf_trampoline_table_t* program_type_specific_helper_trampoline_table;
```

Reply:
Actually I see you added `helper_id` into the trampoline table entry, so why do we need two tables instead of one combined one?   Two make sense if the helper id is just an index into each one and each table is relative to a different base id, in which case you don't need the helper_id as a member in each entry.

Reply:
Agree. Changed to 1 table. Earlier I created 2 tables, and then had to modify trampoline entry to also include 'helper_id`, so 2 tables are not needed.